packages/flutter/test/widgets/semantics_debugger_test.dart

```diff
@@ -334,7 +332,7 @@ void main() {
     final Key keyTop = UniqueKey();
     final Key keyBottom = UniqueKey();
 
-    bool valueTop = false;
+    bool? valueTop = false;
```

Comment:
Can't this variable just be removed? It doesn't appear to be used anywhere, other than to initialize the value. Maybe I'm missing something, though.

Reply:
It's used later by the test, but I also did some cleaning up since some of the test code seems a little odd